### PR TITLE
Fixes (some) clippy warnings

### PR DESCRIPTION
**Pull request by EverlastingBugstopper**
There are still 2 warnings

```console
$ cargo clippy
   Compiling wrangler v1.2.0 (/Users/averyharnish/Documents/work/wrangler)
warning: this function has too many arguments (8/7)
  --> src/commands/kv/key/put.rs:17:1
   |
17 | / pub fn put(
18 | |     target: &Target,
19 | |     user: GlobalUser,
20 | |     id: &str,
...  |
25 | |     expiration_ttl: Option<&str>,
26 | | ) -> Result<(), failure::Error> {
   | |_______________________________^
   |
   = note: #[warn(clippy::too_many_arguments)] on by default
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#too_many_arguments

warning: the function has a cognitive complexity of 36
   --> src/main.rs:45:1
    |
45  | / fn run() -> Result<(), failure::Error> {
46  | |     // Define commonly used arguments and arg groups up front for consistency
47  | |     // The args below are for KV Subcommands
48  | |     let kv_binding_arg = Arg::with_name("binding")
...   |
596 | |     Ok(())
597 | | }
    | |_^
    |
    = note: #[warn(clippy::cognitive_complexity)] on by default
    = help: you could split it up into multiple smaller functions
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#cognitive_complexity
```

### CODE REVIEW

**Comment by EverlastingBugstopper** [comment]
Help would be gr8 for getting these last two warnings fixed up @gabbifish

File: src/commands/kv/key/put.rs [hunk 1]
```diff
@@ -33,28 +33,27 @@ pub fn put(
...
     let mut query_params: Vec<(&str, &str)> = vec![];
-    match expiration {
-        Some(exp) => query_params.push(("expiration", exp)),
-        None => (),
-    }
-    match expiration_ttl {
-        Some(ttl) => query_params.push(("expiration_ttl", ttl)),
-        None => (),
-    }
+    if let Some(exp) = expiration {
```
Comment (gabbifish) [inline src/commands/kv/key/put.rs]: Nice!

File: src/commands/kv/mod.rs [hunk 2]
```diff
@@ -104,9 +104,8 @@ fn url_encode_key(key: &str) -> String {
 // For handling cases where the API gateway returns errors via HTTP status codes
 // (no KV error code is given).
 fn give_status_code_context(status_code: StatusCode) {
-    match status_code {
-        StatusCode::PAYLOAD_TOO_LARGE => message::warn("Returned status code 413, Payload Too Large. Make sure your upload is less than 100MB in size"),
-        _ => (),
+    if let StatusCode::PAYLOAD_TOO_LARGE = status_code {
```
Comment (gabbifish) [inline src/commands/kv/mod.rs]: Can we use `if let` like this? We could honestly just do `if StatusCode::.. = status_code` for now.

Comment (EverlastingBugstopper) [reply src/commands/kv/mod.rs]: This was recommended to me by the compiler. Seems to work fine (merged this change with `feat-kv-additions` locally to test)

```console
$ wrangler kv:bucket upload --binding KV ./static
🌀  Parsing favicon.ico...
🌀  Parsing home.html...
🌀  Parsing .DS_Store...
🌀  Parsing amp/750kb.html...
🌀  Parsing amp/2mb.html...
🌀  Parsing amp/blog.html...
🌀  Parsing amp/home.html...
🌀  Parsing amp/.DS_Store...
🌀  Parsing amp/900kb.html...
🌀  Parsing amp/500kb.html...
🌀  Parsing amp/4mb.html...
🌀  Parsing amp/625kb.html...
🌀  Parsing amp/home2.html...
🌀  Parsing amp/1mb.html...
🌀  Parsing amp/825kb.html...
🌀  Parsing amp/flowerblog.html...
🌀  Parsing sitemap.xml...
🌀  Parsing robots.txt...
⚠️  Returned status code 413, Payload Too Large. Make sure your upload is less than 100MB in size
⚠️  Error 10024: could not put value for key "amp/4mb.html": value length of 4002377 exceeds limit of 2097152
🕵️  See documentation
```

Comment (EverlastingBugstopper) [reply src/commands/kv/mod.rs]: wait that's not the same path jk ignore this comment

**Pull request merged**